src/main/scala/org/scalameter/json/package.scala

```diff
@@ -0,0 +1,14 @@
+package org.scalameter
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+
+
+package object json {
+  private[scalameter] lazy val mapper = {
```

Comment:
Please add a comment here that explains what the `mapper` is and how it is useful. Consider a more descriptive name.

Also, this should be in the `persistence` package, or a subpackage of the `persistence` package.